diff --git a/out/browser.js b/out/browser.js
index f8e667edf726aec063ca377fdd86e95238e6228a..dddf2aa104a54950782abc61bc2f365405c59424 100644
--- a/out/browser.js
+++ b/out/browser.js
@@ -127,6 +127,9 @@ class VSBrowser {
         fs.writeJSONSync(path.join(userSettings, 'settings.json'), defaultSettings);
         console.log(`Writing code settings to ${path.join(userSettings, 'settings.json')}`);
         const args = ['--no-sandbox', '--disable-dev-shm-usage', `--user-data-dir=${path.join(this.storagePath, 'settings')}`];
+        if (process.env.VSCODE_OPEN_WORKSPACE) {
+            args.push(process.env.VSCODE_OPEN_WORKSPACE);
+        }
         if (this.extensionsFolder) {
             args.push(`--extensions-dir=${this.extensionsFolder}`);
         }
